.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@6.1.0
+  python: circleci/python@2.1.1
+
+jobs:
+  plan:
+    executor: python/default
+    steps:
+      - checkout
+      - run: 
+          name: Plan Module Updates
+          command: | 
+            pip install PyGithub
+            ls 
+            ls scripts 
+            echo "Now running py script"
+            python3 scripts/plan.py
+            ls
+            ls scripts
+            cat scripts/moduleOutput.txt
+      - persist_to_workspace:
+          root: ~/project
+          # root: .
+          paths: 
+            - scripts/*.txt
+  
+  build:
+  # TODO Implement
+    executor: node/default
+    steps:
+     - checkout
+     - attach_workspace:
+        at: .
+     - node/install-packages:
+        pkg-manager: npm
+     - run: 
+        command: |
+          ls
+          ls scripts
+          sudo npm install uglify-js -g
+          # cat /workspace/scripts/moduleOutput.txt
+          node create.js scripts/moduleOutput.txt
+      
+
+workflows:
+  continuous-integration:
+    jobs:
+    - plan:
+        context: [Github]
+    - build:
+        requires: [plan]
+    # - deploy:
```

Comment:
This will be the next item we work on, we will upload the compiled files from the build job and sync them with the S3 bucket on AWS. We can create a bucket in staging to test this also.